### PR TITLE
[Dark Mode] Update: Route and directions

### PR DESCRIPTION
**Pull request by vquach2404**
This PR updates the Route and directions category to support dark mode.

### CODE REVIEW

**Review by yo1995 — APPROVED** [review]
On https://github.com/Esri/arcgis-runtime-samples-ios/pull/1004/files#diff-3b85159e6398d1daf141181e7cad6fbe0bd88dcf8f9a3a2f56e82be5b026144bR29 line 29 of rerouting sample, please replace the clear color with default. 

Other than that, all changes look good. 👍

File: arcgis-ios-sdk-samples/Route and directions/Offline routing/OfflineRouting.storyboard [hunk 1]
```diff
@@ -76,7 +74,8 @@
                                 </items>
                             </toolbar>
                         </subviews>
-                        <color key="backgroundColor" systemColor="systemBackgroundColor" cocoaTouchSystemColor="whiteColor"/>
+                        <viewLayoutGuide key="safeArea" id="4Sz-d2-GIP"/>
+                        <color key="backgroundColor" systemColor="tertiarySystemBackgroundColor"/>
                         <constraints>
                             <constraint firstItem="Ohl-3X-6nD" firstAttribute="top" secondItem="msR-U7-cd1" secondAttribute="bottom" constant="-36" id="00W-hv-SbS"/>
```
Comment (yo1995) [inline arcgis-ios-sdk-samples/Route and directions/Offline routing/OfflineRouting.storyboard:80]: It seems the constraint here is not correct. Could you also fix it? Thanks.

Comment (vquach2404) [reply arcgis-ios-sdk-samples/Route and directions/Offline routing/OfflineRouting.storyboard:80]: I've made some adjustments. Please let me know what you think, thanks!

**Review by yo1995 — APPROVED** [review]

File: arcgis-ios-sdk-samples/Route and directions/Offline routing/OfflineRouting.storyboard (hunk 1 above)
Comment (yo1995) [reply arcgis-ios-sdk-samples/Route and directions/Offline routing/OfflineRouting.storyboard:80]: My apologies, it actually worked correctly. 😅 
But the new constraints also work fine!

**Review by saratchandrakarumuri — APPROVED** [review]
Changes are good

**Review by philium — APPROVED** [review]

**Pull request merged**